miasm2/analysis/depgraph.py

```diff
@@ -601,7 +620,7 @@ def get(self, label, elements, line_nb, heads):
             if (not state.pending or
                     state.label in heads or
                     not self._ira.graph.predecessors(state.label)):
-                yield dpResultcls(state, self._ira)
+                yield dpResultcls(state, self._ira, label, line_nb)
```

Comment:
To my opinion, it seems nicer to have the asking input rather than only a few information on it in argument.
By the way, it seems that the .inputs property is just used to propagate the initial state elements, and is only used in DependencyResult. So it can be removed (for simpler and lighter dependency state), and the `initial_state.elements` can be used instead.